sdk/core/azure-core-test/src/main/java/com/azure/core/test/InterceptorManager.java

```diff
@@ -356,8 +354,9 @@ public void close() {
             if (testProxyEnabled) {
                 testProxyRecordPolicy.stopRecording(proxyVariableQueue);
             } else {
-                try (BufferedWriter writer = Files.newBufferedWriter(createRecordFile(playbackRecordName).toPath())) {
-                    RECORD_MAPPER.writeValue(writer, recordedData);
+                try (BufferedWriter writer = Files.newBufferedWriter(createRecordFile(playbackRecordName).toPath());
```

Comment:
I think in longer run we should be good to make a breaking change and remove older APIs before test proxy. Now that we have the migration done.